app/src/main/res/layout/browser_tabstray_item.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="72dp">
+
+    <ImageView
+        android:id="@+id/mozac_browser_tabstray_icon"
+        android:layout_width="100dp"
+        android:layout_height="56dp"
+        android:visibility="gone"
```

Comment:
This is a hack that we should fix by making the view optional in the `TabViewHolder`.

Reply:
Gonna make this as a follow up since we have https://github.com/mozilla-mobile/android-components/pull/6611 attempting to land in the meantime.